Makefile

```diff
@@ -110,7 +110,7 @@ $(FC_TEST_DATA_PATH)/root-drive-ssh-key $(FC_TEST_DATA_PATH)/root-drive-with-ssh
 ifeq ($(GID), 0)
 	$(MAKE) $(FIRECRACKER_DIR)
 	$(FIRECRACKER_DIR)/tools/devtool build_rootfs -m $(FC_TEST_DATA_PATH)/mnt
-	cp $(FIRECRACKER_DIR)/build/rootfs/bionic.rootfs.ext4 $(FC_TEST_DATA_PATH)/root-drive-with-ssh.img
+	cp $(FIRECRACKER_DIR)/build/rootfs/*.rootfs.ext4 $(FC_TEST_DATA_PATH)/root-drive-with-ssh.img
```

Comment:
This looks good for now. We can also make a change to `/tools/devtool` to build the rootfs drive with a custom name provided as an argument.

Reply:
The tricky part is the devtool is owned by [firecracker](https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L1160). We need to request changes upstream to do that.

Reply:
Anyway,  decided to bypass the devtool which seems unstable, and uses pre-built rootfs instead following suggestion from #418.